kustomize/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ./binding-apiserver
+- ./apiserver
```

Comment:
apiserver and binding-apiserver cannot be deployed at the same time, we don't need to install `binding-apiserver` here.